src/components/app/data/hooks/useSearchCatalogs.test.js

```diff
@@ -1,33 +1,40 @@
 import { renderHook } from '@testing-library/react-hooks';
 import '@testing-library/jest-dom/extend-expect';
+import { AppContext } from '@edx/frontend-platform/react';
 import useSearchCatalogs from './useSearchCatalogs';
 import { LICENSE_STATUS } from '../../../enterprise-user-subsidy/data/constants';
 import { features } from '../../../../config';
 import {
-  useCatalogsForSubsidyRequests,
-  useCouponCodes,
   useEnterpriseCustomer,
-  useEnterpriseOffers,
-  useRedeemablePolicies,
-  useSubscriptions,
 } from '..';
-import { enterpriseCustomerFactory } from '../services/data/__factories__';
+import { authenticatedUserFactory, enterpriseCustomerFactory } from '../services/data/__factories__';
+import useCatalogsForSubsidyRequests from './useCatalogsForSubsidyRequests';
+import useSubscriptions from './useSubscriptions';
+import useRedeemablePolicies from './useRedeemablePolicies';
+import useCouponCodes from './useCouponCodes';
+import useEnterpriseOffers from './useEnterpriseOffers';
 
 jest.mock('../../../app/data', () => ({
   ...jest.requireActual('../../../app/data'),
   useEnterpriseCustomer: jest.fn(),
```

Comment:
[curious] Why didn't `useEnterpriseCustomer` need to change here like all the other hooks? On a quick look, it seems like maybe the `useEnterpriseCustomer` isn't actually used by `useSearchCatalogs` anymore, so maybe it could be removed?